modules/RTC/TraceablePeerConnection.js

```diff
@@ -680,6 +680,33 @@ TraceablePeerConnection.prototype.getAudioLevels = function(speakerList = []) {
     return audioLevels;
 };
 
+/**
+ * Checks if the browser is currently doing true simulcast where in three different media streams are being sent to the
+ * bridge. Currently this happens only when VP8 is the selected codec.
+ * @returns {boolean}
+ */
+TraceablePeerConnection.prototype.doesTrueSimulcast = function() {
```

Comment:
I am afraid there is no way to use feature detection here since it will be depend on our implementation. If we choose `L3T3` scalability mode, VP9/AV1 will continue to do SVC and will do true simulcast if we choose `L1T3` as a scalability mode and set `scaleResolutionDownBy` values on the encodings. I can update this method when we implement simulcast for VP9/AV1.

Reply:
Sounds good!